build.gradle

```diff
@@ -22,7 +22,7 @@ apply plugin: 'maven-publish'
 apply plugin: 'com.jfrog.bintray'
 
 group 'uk.gov.hmcts.reform'
-version '4.0.2'
+version '4.0.4'
```

Comment:
shouldn't this be 3

Reply:
done

Reply:
As backwards incompatible changes are introduced, increment the X part here please